### PR TITLE
Card Quality of Life and Hanafuda!

### PR DESCRIPTION
**Pull request by AwkwardDryad**
[FEATURE][QOL]

## About the PR 
- you can now place cards on the floor like you can on tables
- you can stack cards on disarm intent solitaire style
- added a missing sprite for the red deck box
- changed the way reading cards in hand outputs to chat to make it more readable
- adds hanafuda, an art-based tabletop gambling game involving matching cards of the same season and scoring specific combos. (See icon diffs for pretty pictures :D)

## Why's this needed? 
- playing cards are a little frustrating to use, and they need a little love and support :heart: 

## Changelog

```
(u)AwkwardDryad:
(*)Added hanafuda! You can purchase it at a card vendor for 298 credits
(+)You can now place cards on floor tiles in the same manner as on a table
(+)Cards have new solitaire stacking functionality. Check playing cards tips and tricks for more information
```

### CODE REVIEW

**Comment by Enfaeutchie** [comment]
Who's up for Koi-Koi?

**Comment by AwkwardDryad** [comment]
> Who's up for Koi-Koi?

memememememememememe

File: code/obj/item/playing_cards.dm [hunk 1]
```diff
@@ -22,8 +22,9 @@
 	var/card_tappable = 1 // tap 2 islands for mana
 	var/card_tapped = 0 // summon Fog Bank, laugh
 	var/card_spooky = 0
+	var/solitaire_offset = 3 //DEV - change later
```
Comment (ZeWaka) [inline code/obj/item/playing_cards.dm:25]: foo

Comment (AwkwardDryad) [reply code/obj/item/playing_cards.dm:25]: FACK. i forgot to kill the comment

**Pull request merged**